submissions/israelOladosu/DomManipulation/index.js

```diff
@@ -0,0 +1,58 @@
+let submitForm = document.getElementById("validateForm"),
+  qty = document.getElementById("qty"),
+  shipping_cost = document.getElementById("shipping_cost"),
+  total_cost = document.getElementById("total_cost"),
+  price = document.getElementById("price");
+
+const RegEx = /^[0-9]$/;
+
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
I think this `DOMContentLoaded` will only fire once. I think you should still work with the `input` event. So everything will be calculated in real-time as your input your value

Reply:
Oh! Is the shipping cost based on the total cost or the price of a single item?

Reply:
If yes, this will mean my shipping cost will be empty when Dom loads and will display when qty has been entered ryt?